cloud/ec2.go

```diff
@@ -1187,12 +1187,7 @@ func (m *ec2Manager) CostForDuration(ctx context.Context, h *host.Host, start, e
 	if end.Before(start) || util.IsZeroTime(start) || util.IsZeroTime(end) {
 		return 0, errors.New("task timing data is malformed")
 	}
-	ec2Settings := &EC2ProviderSettings{}
-	err := ec2Settings.fromDistroSettings(h.Distro)
```

Comment:
What is the effect of removing calls to `fromDistroSettings`?

Reply:
Resolved in person.